ga-ccd-definition/FixedLists/HearingChannel-GAR3CP-nonprod.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "ID": "HearingChannel",
```

Comment:
Could we reuse the existing one?

Reply:
Created a new JIRA task for refactoring https://tools.hmcts.net/jira/browse/CIV-6823